util/test/demos/vk/vk_shader_debug_zoo.cpp

```diff
@@ -1576,6 +1578,17 @@ void main()
       Color += vec4(1.0, 1.0, 1.0, 1.0);
       break;
     }
+    case 177:
+    {
+      Color = vec4(imageLoad(texBufferUint, int(zeroi+2)).x, 1, 0, 1);
```

Comment:
Minor comment : this test does pass without the code change purely because the value at the offset is 0 and 0 is also returned by the failure case.

Locally this change makes this test fail, by reading a non-zero value

> git diff
diff --git a/util/test/demos/vk/vk_shader_debug_zoo.cpp b/util/test/demos/vk/vk_shader_debug_zoo.cpp
index a19996cd5..f9d2058ff 100644
--- a/util/test/demos/vk/vk_shader_debug_zoo.cpp
+++ b/util/test/demos/vk/vk_shader_debug_zoo.cpp
@@ -1580,7 +1580,7 @@ void main()
     }
     case 177:
     {
-      Color = vec4(imageLoad(texBufferUint, int(zeroi+2)).x, 1, 0, 1);
+      Color = vec4(imageLoad(texBufferUint, int(zeroi+31*4)).x, 1, 0, 1);
       break;
     }
     case 178:
@@ -4552,6 +4552,10 @@ OpMemberDecorate %cbuffer_struct 17 Offset 216    ; double doublePackSource
     memcpy(&cbufferdata[13].x, &unpackDouble, sizeof(unpackDouble));
     memcpy(&cbufferdata[14].z, &unpackDouble, sizeof(unpackDouble));

+    // ubufferimage load/store
+    uint32_t uintValue = 123;
+    memcpy(&cbufferdata[15].x, &uintValue, sizeof(uintValue));
+
     // move to account for offset